### PR TITLE
As 1034 rename  source code UI controller

### PR DESCRIPTION
**Pull request by mlikasam-askui**

### CODE REVIEW

File: packages/askui-nodejs/src/execution/control-your-ui-client.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ export class ControlYourUiClient {
     value: any,
   ) => void = ControlYourUiClient.EMPTY_RESOLVE;
```
Comment (thinh-askui) [inline packages/askui-nodejs/src/execution/control-your-ui-client.ts:45]: Just a question: controlyourui is outdated. Maybe should also rename controlyouruiclient?

**Review by thinh-askui — CHANGES_REQUESTED** [review]

**Review by thinh-askui — APPROVED** [review]

**Review by adi-wan-askui — APPROVED** [review]
Good work :)

File: packages/askui-nodejs/src/execution/client-interface.ts [hunk 2]
```diff
@@ -2,14 +2,14 @@ import { CredentialArgs } from '../utils/http/credentials';
 import { AnnotationLevel } from './annotation-level';
 
 export interface ClientArgs {
-  readonly controlServerUrl?: string,
-  readonly controlYourUiApi?: string,
+  readonly askuiUiControllerServerUrl?: string,
+  readonly inferenceApiApiUrl?: string,
```
Comment (programminx-askui) [inline packages/askui-nodejs/src/execution/client-interface.ts:6]: inferenceServerUrl

File: packages/askui-nodejs/src/lib/download-binaries.ts [hunk 3]
```diff
@@ -11,9 +11,9 @@ enum SupportedPlatform {
 }
 
 const binarySubPathsByPlatform = {
-  linux: ['linux', 'controlui-server.AppImage'],
-  darwin: ['darwin', 'controlui-server.dmg'],
-  win32: ['windows', 'controlui-server.exe'],
+  linux: ['linux', 'askui-ui-controller.AppImage'],
+  darwin: ['darwin', 'askui-ui-controller.dmg'],
+  win32: ['windows', 'askui-ui-controller.exe'],
```
Comment (programminx-askui) [inline packages/askui-nodejs/src/lib/download-binaries.ts:16]: :white_check_mark:

File: packages/askui-nodejs/src/execution/askui-inference-api.ts [hunk 4]
```diff
@@ -7,7 +7,7 @@ import { AnnotationJson } from '../core/annotation/annotation-json';
 import { resizeBase64ImageWithSameRatio } from '../utils/transformations';
 import { IsImageRequired } from './is-image-required-interface';
 
-export class ControlYourUiApi {
+export class InferenceApi {
```
Comment (programminx-askui) [inline packages/askui-nodejs/src/execution/askui-inference-api.ts:10]: InferenceClient

File: examples/askui-nodejs-base-example/Readme.md [hunk 5]
```diff
@@ -22,7 +22,7 @@ parallel:
       alias: <alias_name>
 ```
 
-2 - Write your tests with jasmine. Please note, that you have to use alias_name that you defined in step 1 for the controlServerUrl : `http://<alias_name>:6769`
+2 - Write your tests with jasmine. Please note, that you have to use alias_name that you defined in step 1 for the askuiUiControllerServerUrl : `http://<alias_name>:6769`
```
[the remark this answers is not in the document]
Comment (mlikasam-askui) [reply examples/askui-nodejs-base-example/Readme.md:25]: done

File: docs/docs/general/04-Continuous Integration/askui-ui-controller-docker-images.md [hunk 6]
```diff
@@ -74,34 +74,34 @@ function startTestContainer(): StartedTestContainer {
 let testContainer: StartedTestContainer
 
 // Server for controlling the operating system
-let askuiServer: AskuiControlServer;
+let askuiUiControllerServer: UiControlServer;
```
Comment (programminx-askui) [inline docs/docs/general/04-Continuous Integration/askui-ui-controller-docker-images.md:77]: askuiUiControllerServer --> uiControllerServer

What do you think?

File: packages/askui-nodejs/src/execution/execution-runtime.ts [hunk 7]
```diff
@@ -12,8 +12,8 @@ import { logger } from '../lib/logger';
 
 export class ExecutionRuntime {
   constructor(
-    private client: ControlYourUiClient,
-    private api: ControlYourUiApi,
+    private client: AskuiUiControllerClient,
```
Comment (programminx-askui) [inline packages/askui-nodejs/src/execution/execution-runtime.ts:15]: uiControlClient:

File: packages/askui-nodejs/src/execution/execution-runtime.ts [hunk 8]
```diff
@@ -12,8 +12,8 @@ import { logger } from '../lib/logger';
...
+    private api: InferenceClient,
```
Comment (programminx-askui) [inline packages/askui-nodejs/src/execution/execution-runtime.ts:16]: inferenceClient

File: packages/askui-nodejs/src/execution/askui-ui-controller-client.ts [hunk 9]
```diff
@@ -37,18 +37,18 @@ export class ControlYourUiClient {
 
   private timeout?: NodeJS.Timeout;
 
-  private currentReject: (reason?: unknown) => void = ControlYourUiClient.EMPTY_REJECT;
+  private currentReject: (reason?: unknown) => void = AskuiUiControllerClient.EMPTY_REJECT;
```
[the remark this answers is not in the document]
Comment (mlikasam-askui) [reply packages/askui-nodejs/src/execution/askui-ui-controller-client.ts:40]: we need to agree on one name.

**Comment by programminx-askui** [comment]
Hello @askui/maintainers,

we rename agreement:
- AskuiUIControllerServer -rename-> uiController
- askuiUiControlServerUrl --> uiControllerUrl
- AskuiUIControllerClient -rename-> uiControlClient
- ControlUiApi --> InferenceClient
- remove all prefixes askui*
- rename files remove askui

**Pull request closed without merging**